tests/check/check_signed_video_sign.c

```diff
@@ -498,6 +498,33 @@ START_TEST(correct_signed_stream_with_eos)
 END_TEST
 #endif
 
+START_TEST(check_the_priv_key)
+{
+  SignedVideoCodec codec = settings[_i].codec;
+
+  signed_video_t *sv = signed_video_create(codec);
+  ck_assert(sv);
+  char *private_key = NULL;
+  size_t private_key_size = 0;
+
+  ck_assert(read_test_private_key(settings[_i].ec_key, &private_key, &private_key_size, false));
+  signed_video_set_private_key(sv, private_key, private_key_size);
+  const char *private_key_2 = get_private_key_from_sv(sv);
+  private_key_size = strlen(private_key_2);
```

Comment:
Can we expect the size to be unchanged? If so, I would instead check that with
`ck_assert_int_eq(strlen(reconstructed_private_key), private_key_size);`